src/main/java/org/opensearch/security/rest/SecurityInfoAction.java

```diff
@@ -61,13 +62,11 @@
 import org.opensearch.threadpool.ThreadPool;
 import com.google.common.collect.ImmutableList;
 
-import static org.opensearch.security.dlic.rest.support.Utils.addRoutesPrefix;
-
 public class SecurityInfoAction extends BaseRestHandler {
-    private static final List<Route> routes = addRoutesPrefix(ImmutableList.of(
+    private static final List<ReplacedRoute> replacedRoutes = RestHandler.replaceRoutes(ImmutableList.of(
             new Route(GET, "/authinfo"),
             new Route(POST, "/authinfo")
-    ),"/_opendistro/_security", "/_plugins/_security");
+    ),"/_plugins/_security", "/_opendistro/_security");
```

Comment:
not required, unnecessary change right?

Reply:
It is necessary as  in `ReplacedRoute` method we should pass new prefix first and then the deprecated one:`ReplacedRoute(RestHandler.Route route, String prefix, String deprecatedPrefix)`

Reply:
Got it